Resources/Prototypes/_Sunrise/Catalog/uplink_catalog.yml

```
@@ -32,6 +32,42 @@
+  categories:
+  - UplinkWearables
+  conditions:
+  - !type:StoreWhitelistCondition
+    whitelist:
+      tags:
+      - NukeOpsUplink
+      - LoneOpsUplink
+
+- type: listing
+  id: UplinkPouchExplosive
+  icon: { sprite: /Textures/_RMC14/Objects/Clothing/Pouches/large_explosive.rsi, state: icon }
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)